src/main/resources/config/spark/sample_connections_config.yaml

```diff
@@ -5,8 +5,8 @@ connections:
 - id: spark_0
   driver: org.apache.hive.jdbc.HiveDriver
   url: jdbc:hive2://127.0.0.1:10000
-  username: admin
-  password: p@ssw0rd0
+  username: ${DATABASE_USER:spark_admin}
```

Comment:
The password is set as a variable in the pom (is there a better place to do this?), how is the user set?

Reply:
I have updated it in the test instead. The username is not set, hence it picks the default value mentioned here. I wrote the test in a way that we handle both cases.

Reply:
Maybe add somewhere in the documentation (README?) that setting environmental variables is now an option, otherwise this will be confusing for people reading through the sample configurations for the first time.